storage.go

```diff
@@ -195,6 +205,12 @@ func (s *Storage) move(ctx context.Context, src string, dst string, opt pairStor
 
 	err = os.Rename(rs, rd)
 	if err != nil {
+		if errors.Is(err, os.ErrExist) {
```

Comment:
I tested on Windows 10 and got the syscall.Errno `ERROR_ACCESS_DENIED (5)`. Actually the file and dir can be accessed. I tried to call `os.Chmod` after `os.MkdirAll` and `os.Create`, but it didn't work.